test/spec_helper.rb

```diff
@@ -18,6 +18,8 @@
 
 $:.unshift(File.expand_path("../../src/lib", __FILE__))
 
+require "yast"
```

Comment:
I am nnot sure, but probably line above is no longer needed if you properly set Y2DIR and then require yast